drivers/char/tpm/tpm_tis_spi_main.c

```diff
@@ -232,6 +234,37 @@ static int tpm_tis_spi_probe(struct spi_device *dev)
 	return tpm_tis_spi_init(dev, phy, irq, &tpm_spi_phy_ops);
 }
 
+static int tpm_tis_spi_parse_dt(struct device *dev)
+{
+	int gpio;
+	int rc = 0;
+
+	gpio = of_get_named_gpio_flags(dev->of_node, "reset-gpio", 0, NULL);
```

Comment:
If I see this right the commonly used name is `reset-gpios` even if only one gpio is used.
```
prosenberger@bob:~/src/linux-5.10[devel/ilu/revpi-5.10-base ?]$ git grep -e "reset-gpio " -e reset-gpio: | wc -l
163
prosenberger@bob:~/src/linux-5.10[devel/ilu/revpi-5.10-base ?]$ git grep -e "reset-gpios " -e reset-gpios: | wc -l
1066
```

Reply:
And I'm irritated, that there are many hits for reset-gpios in the Documentation directory but only a few in the code.
```
	ctx->reset = devm_gpiod_get(&spi->dev, "reset", GPIOD_OUT_LOW);
	if (IS_ERR(ctx->reset)) {
		dev_err(&spi->dev, "Couldn't get our reset line\n");
		return PTR_ERR(ctx->reset);
	}
```
This is from the panel-sitronix-st7789v.c the Documentation has this DT snipped:
```
        panel@0 {
            compatible = "sitronix,st7789v";
            reg = <0>;
            reset-gpios = <&pio 6 11 GPIO_ACTIVE_LOW>;
            backlight = <&pwm_bl>;
            ....
```

Reply:
Here it is: https://elixir.bootlin.com/linux/latest/source/drivers/gpio/gpiolib-of.c#L503
It will try to concatenate `-gpios` or `-gpio` to the request string.
```
 	gpio = of_get_named_gpio_flags(dev->of_node, "reset", 0, NULL);
```

Reply:
```suggestion
	reset_gpio = of_get_named_gpio_flags(dev->of_node, "reset", 0, NULL);
```